public/styles.css

```
@@ -0,0 +1,14 @@
+    font-family: 'Circular Std';
+    src: url('/public/assets/fonts/circular-std-medium-500.ttf');
+}
+
+/*animation mixin*/
+.btn-primary {
+    background-color: #5d50c6;
+    color: #eeeeee;
+    border-radius: 6.25rem;
+}
+body {
+    font-family: Circular Std;
```

Biome (1.9.4)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)